Makefile

```diff
@@ -35,7 +30,10 @@ install: manifests
 kustomize:
 	@echo "updating kustomize namespace to ${NAMESPACE}"
 	sed -i'' -e 's@namespace: .*@namespace: '"${NAMESPACE}"'@' ./config/default/kustomization.yaml
-	kustomize build config/default -o ./config/deploy.yaml
```

Comment:
Moving the `kustomize` commands into docker are necessary due to the older version required by kubebuilder v1.

Reply:
Which we need for our production hosting version right...